doc/release-notes/5766-citation-tsv

```diff
@@ -0,0 +1,4 @@
+Update citation metadata block
+curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @citation.tsv -H "C
```

Comment:
@djbrooke this curl command seems incomplete. It ends with ` -H "C`

Reply:
Also, does it work? Is removing a tab character one of the supported operations for reloading a metadata block? I thought the main support operation was adding (but not removing) additional controlled vocabulary. @scolapasta and @sekmiller know better than I do what's supported.

Reply:
Thanks, updated in https://github.com/IQSS/dataverse/pull/6022/commits/253f12d93e90a4c23d7017609e99d204af6296a0

Reply:
I talked with @kcondon about this a bit yesterday and I know he worked with @scolapasta as noted above, so his knowledge has been involved. Will resolve this convo. If the fix for this small changes does not work as expected, we'll revisit.

Reply:
@djbrooke oh! I missed the note at https://github.com/IQSS/dataverse/pull/6022#issuecomment-512999475 ! That's fantastic that this is a supported operation!